### PR TITLE
setup sass processing client

### PR DESCRIPTION
**Pull request by MartinKaburu**
#### What does this PR do?
It integrates SASS into the project.
#### Description of Task to be completed?
Integrate SASS processing into the project codebase.
#### What are the relevant pivotal tracker stories?
[#161255191](https://www.pivotaltracker.com/story/show/161255191)

### CODE REVIEW

**Review by p8ul — APPROVED** [review]
Everything looks cool, sass well integrated

File: src/index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
-import './assets/styles/index.css';
+import './assets/styles/index.sass';
```
Comment (Tony-Ndichu) [inline src/index.js]: Hey, is the extension usually '.sass' or '.scss'

**Review by Mirriam-Maina — APPROVED** [review]
I like the choice to use sass instead of scss(Sassy Css) it is much easier to use and read

File: src/index.js (hunk 1 above)
Comment (mmosoroohh) [reply src/index.js]: The reason we used `sass` was that it's easier and simpler to code plus it doesn't have the curly braces which make you write less code as compared to `scss` which you have to use the curly braces thus making you wite lengthy code.  But I am open to suggestions on which one is better.

Comment (Tony-Ndichu) [reply src/index.js]: Thanks for that, I wasn't aware of that

Comment (Rodgers-M) [reply src/index.js]: @mmosoroohh I agree with you on writing little code, but on the other side, `.scss` is closer to `css` and thus I would prefer this over `.sass` for readability and also familiarity.

File: package.json [hunk 2]
```diff
@@ -37,7 +37,9 @@
     "coveralls": "^3.0.2",
     "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-jsx-a11y": "^6.1.1",
-    "eslint-plugin-react": "^7.11.0"
+    "eslint-plugin-react": "^7.11.0",
+    "node-sass": "^4.10.0",
+    "sass-loader": "^7.1.0"
```
Comment (Rodgers-M) [inline package.json]: i don't think we need `sass-loader` anymore, with `create-react-app` version 2.x sass is supported out of the box and only need `node-sass`

**Review by Rodgers-M — APPROVED** [review]

**Pull request merged**